submissions/Vitaminvp/HTML&CSSPractice/style.css

```diff
@@ -0,0 +1,237 @@
+body {
+    font-family: "Arial", sans-serif;
+    background-color: #fff;
+    overflow-x: hidden;
+}
+ul {
+    margin: 0;
+    padding: 0;
+    list-style-type: none;
+}
+.nav{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+}
+.nav__input,
+.nav__menu,
+.nav__menu_input,
+.nav__menu_listMore{
+    display: none;
+}
+.nav__menu{
+    position: absolute;
+    top: 50px;
+    border: 1px solid #ccc;
+    border-color: rgba(0,0,0,.2);
+    box-shadow: 0 2px 10px rgba(0,0,0,.2);
+    background: #fff;
+    border-radius: 2px;
+    width: 337px;
+    max-height: 634px;
+    height: 496px;
+    overflow-y: overlay;
```

Comment:
Please, let me know what does `overflow-y: overlay;` mean?

Reply:
append scrollbars if necessary and overlay over/above content
https://stackoverflow.com/questions/43050434/what-is-the-function-of-overlay-value-of-overflow-property

Reply:
Try to test your app in Firefox.
This property is supported only in -Webkit browsers and it's also non-standardized.
Would you try another property for `overflow` which will be more cross-browser and universal?
Also, I strongly recommend using [CanIUse](https://caniuse.com/) for testing different properties.

Reply:
Changed to auto.